src/adaXT/predict/predict.pyi

```diff
@@ -84,13 +84,13 @@ class PredictClassification(Predict):
 
 class PredictRegression(Predict):
     """
-    Default used for Regression.
+        The default prediction calls for the Regression tree type.
```

Comment:
The default prediction calls the Regression tree type

Reply:
Updated this entire file